docker-compose-tests.yaml

```diff
@@ -0,0 +1,33 @@
+version: '3.6'
+services:
+  postgres:
+    image: postgres:11.2-alpine
+    restart: always
+    volumes:
+    - db_data:/var/lib/postgresql-test/data
+  graphql-engine:
+    image: hasura/graphql-engine:v1.0.0-beta.8
+    ports:
+    - 9000:8080
```

Comment:
Port 8080 is already taken by jenkins, therefore I map the graphql engine's port 8080 to another arbitrary port. That's only for testing purposes. In production, the engine will serve on port 8080.

Reply:
Yes

Reply:
By the way, the `hasura-service-tests` container is only connected to the `host` network **during the build**, not during execution. To connect that container to the `host` network at container runtime, you need to add a `network_mode: host`, like this:
```
hasura-service-tests:
    build:
      context: ./database-service
      dockerfile: ./Dockerfile
      network: host
      target: unittest
    network_mode: host
    restart: "no" 
    depends_on:
      - graphql-engine
    volumes:
      - ./database-service:/app
```
Try to run the `docker-compose-test.yml` on your dev laptop when port `8080` is already taken.  You will not be able to run the hasura service.